src/content/eduhelx/eduHelxFeatures.mdx

```diff
@@ -0,0 +1,55 @@
+import GridItem from "../../components/sections/features";
+import { GridContainer } from "../../components/grid";
+import { HeadingSection } from "../../components/sections/heading";
+import { MaxWidthWrapper } from "../../components/layout/maxWidthWrapper";
+import customizedIcon from "../../images/eduhelx/customized-icon.png";
+import flexibleIcon from "../../images/eduhelx/flexible-icon.png";
+import launchIcon from "../../images/eduhelx/launch-icon.png";
+import cloudBasedIcon from "../../images/eduhelx/cloud-based-icon.png";
+
+<MaxWidthWrapper backgroundColor="#F5F6F5" gutterBottom>
+
+<HeadingSection>
+## Fantastic Features
+
+EduHelx is customized for the unique needs of your classroom, your materials, and your students.
+
+</HeadingSection>
+
+<GridContainer spacing={4} padding="2rem 0 0 0">
+
+<GridItem
```

Comment:
my comment is similar to the one i left on the `home/helxFeatures.mdx` section. the title and subheading text can be moved into markdown and, since this even uses the same component as the `home/helxFeatures.mdx`, it should be easy to adapt